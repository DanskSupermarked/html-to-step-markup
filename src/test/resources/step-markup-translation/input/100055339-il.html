<p><b>ZyXEL NWD2205 - Ultra Compact Wireless N300 USB Adapter  </b> <br /> Ultra-kompakt design for optimal bærbarhed 802.11n overholdelse dataoverførselshastigheder på op til 300 Mbps * Soft AP funktion til deling af trådløs internetadgang Wi-Fi WMM certificeret for prioritering multimediestreaming One-touch WPS-knappen for nem trådløs sikkerhed opsætning.  Grøn adapter sparer energi ubesværet . <br /> <br /> <b>Specifikationer:</b><il></il></p> <ul>     <li><il>WLAN: Nej</il></li>     <li><il>Trådløse protokoller: IEEE 802.11b/g, IEEE 802.11n</il></li>     <li><il>Kryptering: 64-bit WEP, 128-bit WEP, WPA, WPA2</il></li>     <li><il>LED indikator: Signal styrke, trådløs power, WPS, </il><il>Ethernet</il></li>     <li><il>1 stk. USB 1.1/2.0 port/ tilslutning</il> <il></il></li>     <li><il>Vægt: 3 gram</il> <il></il></li>     <li><il>Mål: 36 mm x 6 mm x 18 mm </il></li> </ul>
