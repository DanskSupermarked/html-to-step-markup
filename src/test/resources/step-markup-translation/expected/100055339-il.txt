<bold>ZyXEL NWD2205 - Ultra Compact Wireless N300 USB Adapter</bold><return/>
Ultra-kompakt design for optimal bærbarhed 802.11n overholdelse dataoverførselshastigheder på op til 300 Mbps * Soft AP funktion til deling af trådløs internetadgang Wi-Fi WMM certificeret for prioritering multimediestreaming One-touch WPS-knappen for nem trådløs sikkerhed opsætning. Grøn adapter sparer energi ubesværet .<return/>
<return/>
<bold>Specifikationer:</bold> <return/>
<bulletlist>
<bullet>WLAN: Nej</bullet>
<bullet>Trådløse protokoller: IEEE 802.11b/g, IEEE 802.11n</bullet>
<bullet>Kryptering: 64-bit WEP, 128-bit WEP, WPA, WPA2</bullet>
<bullet>LED indikator: Signal styrke, trådløs power, WPS,Ethernet</bullet>
<bullet>1 stk. USB 1.1/2.0 port/ tilslutning</bullet>
<bullet>Vægt: 3 gram</bullet>
<bullet>Mål: 36 mm x 6 mm x 18 mm</bullet>
</bulletlist>
